.github/workflows/deploy_preview.yml

```diff
@@ -1,33 +1,38 @@
-# name: Deploy Preview to Netlify
+name: Deploy Preview to Netlify
 
-# on: [pull_request, workflow_dispatch]
+on: pull_request
 
-# jobs:
-#   deploy-preview:
-#     name: Build and Deploy preview on Netlify
-#     runs-on: ubuntu-latest
-#     steps:
-#       - name: Cancel Previous Runs
-#         uses: styfle/cancel-workflow-action@0.10.0
-#         with:
-#           access_token: ${{ github.token }}
+jobs:
+  deploy-preview:
+    name: Build and Deploy preview on Netlify
+    runs-on: ubuntu-latest
+    steps:
+      - name: Cancel Previous Runs
+        uses: styfle/cancel-workflow-action@0.10.0
+        with:
+          access_token: ${{ github.token }}
 
-#       - name: Checkout
-#         uses: actions/checkout@v3
+      - name: Checkout repository
+        uses: actions/checkout@v3.2.0
 
-#       - name: Install modules
-#         run: npm ci
+      - name: Install modules
+        run: npm ci
 
-#       - name: Build
-#         run: npm run build
+      - name: Decode .env file
```

Comment:
can you explain why we need to decode this file 💭

Reply:
@nvminhtue because Github Action doesn't provide the storage to keep the sensitive file like `.env`, so I encode this file and save it into the environment variable of Github Action. Then decode it in CI.
This approach is often used on Mobile side. [Ref](https://docs.codemagic.io/yaml-basic-configuration/configuring-environment-variables/) 🙏 
Do you have any better way to store `.env` file? 😄